server/container/util/src/main/java/org/apache/james/util/FunctionalUtils.java

```diff
@@ -31,6 +31,13 @@
         };
     }
 
+    public static <T> UnaryOperator<T> toFunction(Runnable runnable) {
```

Comment:
you don't really return a Function but a UnaryOperator.
BTW, how can you transform a void method into a Function? It makes little sense without more documentation.

Reply:
Or with better naming: identityWithSideEffect